### DISCUSSION TITLE
我似乎配置不上模板

### DISCUSSION THREAD

**Discussion by nuo0930** [cpeditor/cpeditor-cn#25] [提问]
新建后仍然是空的文件，配置如下 
![image](https://user-images.githubusercontent.com/63394993/217014507-6d709ef1-ec50-4eef-ba29-cda8c43f0b15.png)

**Comment by ouuan** [comment]
可以检查一下路径是否正确，看图好像 `.` 和 `cpp` 之间有个空格？
